k8s-manifests/database/pv.yml

```
@@ -0,0 +1,19 @@
+kind: PersistentVolume
+
+metadata:
+  name: mongo-pv
+  namespace: three-tier  
+
+spec:
+  storageClassName: standard
+
+  capacity:
+    storage: 5Gi
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)